test/lib/recordevents/event_info_store.go

```diff
@@ -223,8 +224,8 @@ func (ei *EventInfoStore) Find(f EventInfoMatcher) ([]EventInfo, SearchedInfo, [
 
 // Assert that there are at least min number of matches of f.
 // This method fails the test if the assert is not fulfilled.
-func (ei *EventInfoStore) AssertAtLeast(min int, f EventInfoMatcher) []EventInfo {
-	events, err := ei.waitAtLeastNMatch(f, min)
+func (ei *EventInfoStore) AssertAtLeast(min int, f ...EventInfoMatcher) []EventInfo {
```

Comment:
I think same as above, the comments in all these methods still presume a single `f`.